<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rizwan-PC" name="Command line test 5b4eabb3-84da-4784-8572-e4d860e30bd9" tests="3" failures="3" timestamp="26 Apr 2021 06:04:04 GMT" time="48.233" errors="0">
  <testcase name="login" time="15.807" classname="org.testing.TestScripts.TC1">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC1.login(TC1.java:40)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.407" classname="org.testing.TestScripts.TC3">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC3.login(TC3.java:26)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.34" classname="org.testing.TestScripts.TC4">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC4.login(TC4.java:27)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Command line test 5b4eabb3-84da-4784-8572-e4d860e30bd9 -->
